fdbclient/NativeAPI.actor.cpp

```diff
@@ -8122,6 +8122,18 @@ ACTOR Future<Version> verifyBlobRangeActor(Reference<DatabaseContext> cx, KeyRan
 	state Version readVersionOut = invalidVersion;
 	state int batchSize = BUGGIFY ? deterministicRandom()->randomInt(2, 10) : CLIENT_KNOBS->BG_TOO_MANY_GRANULES / 2;
 	state int loadSize = (BUGGIFY ? deterministicRandom()->randomInt(1, 20) : 20) * batchSize;
+
+	if (version.present()) {
+		Version v = version.get();
+		if (v == latestVersion) {
+			Version _version = wait(tr.getReadVersion());
```

Comment:
Actually i believe you want this to call to be in a transaction retry loop, because getReadVersion can throw errors when throttled and you want to tr.onError() that.

Reply:
Simulation would probably catch this if you changed the BlobGranuleRangesWorkload to sometimes call with latest version